hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenFeeScheduleUpdateHandler.java

```diff
@@ -96,7 +96,7 @@ public void handle(
         requireNonNull(tokenStore);
 
         // get the latest configuration
-        final var config = context.getConfiguration().getConfigData(TokenServiceConfig.class);
+        final var config = context.getConfiguration().getConfigData(TokensConfig.class);
```

Comment:
We should discuss if the config records can be placed in the service modules (api, not impl modules)

Reply:
Yes, that will be ideal. Thought of it, but it can be done at a later stage too.